src/click-processor.cr

```diff
@@ -28,6 +29,27 @@ OptionParser.parse do |parser|
   parser.on("-c", "--concatenate", "Concatenate the files") { operations["concatenate"] = true }
   parser.on("-m", "--mix", "Mix the files") { operations["mix"] = true }
   parser.on("-l", "--level", "Level the audio files") { operations["level"] = true }
+  parser.on("-x", "--demo", "Run In Demo Mode") { demo = true }
+end
+
+if demo
+  intro = File.new(directory.chomp("/") + "/intro.m4a").path
+  caleb = File.new(directory.chomp("/") + "/caleb.m4a").path
+  daniel = File.new(directory.chomp("/") + "/daniel.m4a").path
+  outro = File.new(directory.chomp("/") + "/outro.m4a").path
```

Comment:
I don't think you need to open these files to just to get their paths. I think path just returns the input to File.new. I think you can say:

`intro = directory.chomp("/") + "/intro.m4a"`

Reply:
Infact, I think:

`intro = Path.[directory].join("intro.m4a").to_s`

might be a little better. I have this running on my local machine. Idk how to open source, so idk how I can push that up 😞